src/test/kotlin/com/noumenadigital/npl/cli/TestUtils.kt

```diff
@@ -17,7 +17,18 @@ object TestUtils {
     private fun getNplPath(): String {
         // For running with the binary
         val rootDir = File(".").canonicalFile
-        return rootDir.resolve("target/npl").absolutePath
+        val targetDir = rootDir.resolve("target")
+
+        val candidates =
+            targetDir.listFiles { _, name ->
+                name == "npl" || name.startsWith("npl-")
+            }
+
+        if (!candidates.isNullOrEmpty()) {
+            return candidates.maxByOrNull(File::lastModified)?.absolutePath!!
+        }
+
+        throw IllegalStateException("Cannot locate NPL native binary in ${targetDir.absolutePath}")
```

Comment:
Because we had to remove the symlink above https://github.com/NoumenaDigital/npl-cli/pull/24/files#r2104677381